fugue-deploy/src/main/java/com/symphony/oss/fugue/deploy/ArtifactoryHelper.java

```diff
@@ -0,0 +1,98 @@
+/*
+ *
+ *
+ * Copyright 2020 Symphony Communication Services, LLC.
```

Comment:
Create a new class and copy the copyright block from there then delete the new class